[BUG] Test error windows gpu with asyncio

### Description
<!--- Describe your issue/bug/request in detail -->
```

tests\smoke\test_deeprec_model.py ...                                    [ 30%]
tests\smoke\test_notebooks_gpu.py .FFFFFF                                [100%]

================================== FAILURES ===================================
_______________________________ test_ncf_smoke ________________________________

notebooks = {'als_deep_dive': 'C:\\Users\\recocat\\agent\\_work\\26\\s\\notebooks\\02_model\\als_deep_dive.ipynb', 'als_pyspark': ...cornac_bpr_deep_dive': 'C:\\Users\\recocat\\agent\\_work\\26\\s\\notebooks\\02_model\\cornac_bpr_deep_dive.ipynb', ...}

    @pytest.mark.smoke
    @pytest.mark.gpu
    def test_ncf_smoke(notebooks):
        notebook_path = notebooks["ncf"]
        pm.execute_notebook(
            notebook_path,
            OUTPUT_NOTEBOOK,
            kernel_name=KERNEL_NAME,
>           parameters=dict(TOP_K=10, MOVIELENS_DATA_SIZE="100k", EPOCHS=1, BATCH_SIZE=256),
        )

tests\smoke\test_notebooks_gpu.py:29: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\papermill\execute.py:96: in execute_notebook
    start_timeout=start_timeout,
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\papermill\engines.py:56: in execute_notebook_with_engine

tests\integration\test_notebooks_gpu.py:250: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\papermill\execute.py:96: in execute_notebook
    start_timeout=start_timeout,
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\papermill\engines.py:56: in execute_notebook_with_engine
    return self.get_engine(engine_name).execute_notebook(nb, kernel_name, **kwargs)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\papermill\engines.py:296: in execute_notebook
    nb = cls.execute_managed_notebook(nb_man, kernel_name, log_output=log_output, **kwargs)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\papermill\engines.py:352: in execute_managed_notebook
    preprocessor.preprocess(nb_man, kwargs)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\papermill\preprocess.py:24: in preprocess
    with self.setup_preprocessor(nb_man.nb, resources, km=km):
C:\Anaconda\envs\nightly_reco_gpu\lib\contextlib.py:81: in __enter__
    return next(self.gen)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\nbconvert\preprocessors\execute.py:324: in setup_preprocessor
    self.km, self.kc = self.start_new_kernel(cwd=path)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\papermill\preprocess.py:37: in start_new_kernel
    km, kc = super(PapermillExecutePreprocessor, self).start_new_kernel(**kwargs)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\nbconvert\preprocessors\execute.py:270: in start_new_kernel
    config=self.config)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\traitlets\traitlets.py:958: in __new__
    inst.setup_instance(*args, **kwargs)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\traitlets\traitlets.py:986: in setup_instance
    super(HasTraits, self).setup_instance(*args, **kwargs)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\traitlets\traitlets.py:977: in setup_instance
    value.instance_init(self)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\traitlets\traitlets.py:1691: in instance_init
    self._resolve_classes()
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\traitlets\traitlets.py:1696: in _resolve_classes
    self.klass = self._resolve_string(self.klass)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\traitlets\traitlets.py:1507: in _resolve_string
    return import_item(string)
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\traitlets\utils\importstring.py:34: in import_item
    module = __import__(package, fromlist=[obj])


C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\zmq\eventloop\__init__.py:3: in <module>
    from zmq.eventloop.ioloop import IOLoop
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\zmq\eventloop\ioloop.py:78: in <module>
    while _IOLoop.configurable_default() is not _IOLoop:
C:\Anaconda\envs\nightly_reco_gpu\lib\site-packages\tornado\ioloop.py:328: in configurable_default
    from tornado.platform.asyncio import AsyncIOLoop
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    """
    
    import concurrent.futures
    import functools
    
    from threading import get_ident
    from tornado.gen import convert_yielded
    from tornado.ioloop import IOLoop, _Selectable
    
    import asyncio
    
    import sys
    
    if sys.platform == 'win32':
>       asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
E       AttributeError: module 'asyncio' has no attribute 'WindowsSelectorEventLoopPolicy'


```

link to full logs: https://dev.azure.com/best-practices/recommenders/_build/results?buildId=22629&view=logs&j=513fa82f-6bf5-5d01-2457-cb22781e601b&t=7a5e6e9e-f946-529b-9808-e051053a681e

### In which platform does it happen?
<!--- Describe the platform where the issue is happening (use a list if needed) -->
<!--- For example: -->
Azure Data Science Virtual Machine windows
<!--- * Azure Databricks.  -->
<!--- * Other platforms.  -->

### How do we replicate the issue?
<!--- Please be specific as possible (use a list if needed). -->
<!--- For example: -->
<!--- * Create a conda environment for pyspark -->
<!--- * Run unit test `test_sar_pyspark.py` with `pytest -m 'spark'` -->
<!--- * ... -->

### Expected behavior (i.e. solution)
<!--- For example:  -->
<!--- * The tests for SAR PySpark should pass successfully. -->

### Other Comments

--- a/scripts/generate_conda_file.py
+++ b/scripts/generate_conda_file.py
@@ -35,6 +35,7 @@ $ python -m ipykernel install --user --name {conda_env} --display-name "Python (
 CHANNELS = ["defaults", "conda-forge", "pytorch", "fastai"]
 
 CONDA_BASE = {
+    "python": "python==3.6.10",
     "bottleneck": "bottleneck==1.2.1",
     "dask": "dask>=0.17.1",
     "fastparquet": "fastparquet>=0.1.6",
@@ -43,9 +44,8 @@ CONDA_BASE = {
     "matplotlib": "matplotlib>=2.2.2",
     "mock": "mock==2.0.0",
     "numpy": "numpy>=1.13.3",
-    "pandas": "pandas>=0.23.4",
+    "pandas": "pandas>=0.23.4,<1.0.0",
     "pip": "pip>=19.0.3",
-    "python": "python==3.6.8",
     "pytest": "pytest>=3.6.4",
     "pytorch": "pytorch-cpu>=1.0.0",
     "seaborn": "seaborn>=0.8.1",

